metadata-ws/src/test/resources/features/release-date.feature

```diff
@@ -118,12 +118,13 @@ Feature: Transitive release date control for child objects of a Study
 
     # Set release dates for studies according to the scenario
     When I request PATCH STUDY1 with patch and day <S1_RELEASE>
-    Then the response code should be 200
+    Then the response code should be 2xx
```

Comment:
As Sreenidhi mentioned elsewhere, it is better to check for specific status codes rather than for the general ones. Is there any reason to change from `200` to `2xx`?

Reply:
200 and 204 are just successful with ok and no content so I wanted to show that it is successful.  but 404 and 403 might be surely needed to be differntiate as Sreenidhi mentioned.

Reply:
OK, good point, agreed